acquisition_localizer_single.py

```diff
@@ -245,11 +245,12 @@ def resolve_s1_slc(identifier, download_url, asf_queue, esa_queue):
     #asf_queue = "spyddder-sling-extract-asf"
     #esa_queue = "spyddder-sling-extract-scihub"
 
-    # determine best url and corresponding queue
+    # determine best url and corresponding queue by getting first 100 bytes
     vertex_url = "https://datapool.asf.alaska.edu/SLC/SA/{}.zip".format(identifier)
-    r = requests.head(vertex_url, allow_redirects=True)
+    headers = {"Range": "bytes=0-100"}
+    r = requests.get(vertex_url, allow_redirects=True, headers=headers)
     logger.info("Status Code from ASF : %s" %r.status_code)
-    if asf_queue.upper() != "NA" and r.status_code in (200, 403):
+    if asf_queue.upper() != "NA" and r.status_code in (200, 206):
         url = r.url
```

Comment:
206 will be returned if partial SLC data is read by requests, 404 will be returned if SLC data does not exist: https://gist.github.com/shitong01/302f2d04c0f9264d6d7f0df6b6a030d7